src/Sylius/Component/Resource/Model/ToggleableTrait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Model;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+trait ToggleableTrait
+{
+    /**
+     * @var bool
+     */
+    protected $enabled = true;
+
+    /**
+     * @return bool
+     */
+    public function isEnabled()
+    {
+        return $this->enabled;
+    }
+
+    /**
+     * @param bool $enabled
+     */
+    public function setEnabled($enabled)
+    {
+        $this->enabled = (bool) $enabled;
```

Comment:
What about fluent interfacing? Have a policy for that? Otherwise returning the object would be great.